src/sentry/search/events/builder/metrics.py

```diff
@@ -376,14 +377,51 @@ def validate_aggregate_arguments(self) -> None:
         if not self.use_metrics_layer:
             super().validate_aggregate_arguments()
 
+    # This property is used to determine if a query is using at least one of the fields in the spans namespace.
+    @property
+    def is_spans_metrics_query(self) -> bool:
```

Comment:
nit can you make these docstrings for both functions
```suggestion
    @property
    def is_spans_metrics_query(self) -> bool:
    """This property is used to determine if a query is using at least one of the fields in the spans namespace."""
```